spec/requests/api/sessions_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Sessions', type: :request do
+  let(:user) { FactoryBot.create(:user) }
```

Comment:
Same thing here. You can omit FactoryBot.

Reply:
Fixed